app/src/main/java/app/revanced/manager/ui/screen/settings/AboutSettingsScreen.kt

```diff
@@ -37,42 +36,60 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import app.revanced.manager.BuildConfig
 import app.revanced.manager.R
+import app.revanced.manager.network.dto.ReVancedSocial
 import app.revanced.manager.ui.component.AppTopBar
+import app.revanced.manager.ui.viewmodel.AboutViewModel
 import app.revanced.manager.ui.component.settings.SettingsListItem
 import app.revanced.manager.util.isDebuggable
 import app.revanced.manager.util.openUrl
 import com.google.accompanist.drawablepainter.rememberDrawablePainter
+import compose.icons.FontAwesomeIcons
+import compose.icons.fontawesomeicons.Brands
+import compose.icons.fontawesomeicons.brands.Discord
+import compose.icons.fontawesomeicons.brands.Github
+import compose.icons.fontawesomeicons.brands.Reddit
+import compose.icons.fontawesomeicons.brands.Telegram
+import compose.icons.fontawesomeicons.brands.Twitter
+import compose.icons.fontawesomeicons.brands.Youtube
+import org.koin.androidx.compose.getViewModel
 
 @OptIn(ExperimentalMaterial3Api::class, ExperimentalLayoutApi::class)
 @Composable
 fun AboutSettingsScreen(
     onBackClick: () -> Unit,
     onContributorsClick: () -> Unit,
     onLicensesClick: () -> Unit,
+    viewModel: AboutViewModel = getViewModel()
 ) {
     val context = LocalContext.current
     // painterResource() is broken on release builds for some reason.
     val icon = rememberDrawablePainter(drawable = remember {
         AppCompatResources.getDrawable(context, R.drawable.ic_logo_ring)
     })
 
-    val filledButton = listOf(
-        Triple(Icons.Outlined.FavoriteBorder, stringResource(R.string.donate)) {
-            context.openUrl("https://revanced.app/donate")
-        },
-        Triple(Icons.Outlined.Language, stringResource(R.string.website), third = {
-            context.openUrl("https://revanced.app")
-        }),
+    val socialIcons = mapOf(
```

Comment:
This should be a global variable or part of a companion object.

Reply:
If I were to make `socialIcons` a global variable, where should I declare it?